README.md

```diff
@@ -1114,6 +1166,56 @@ func main() {
 
 Caveats
 -------
+### Exec Mode
+
+As of version 0.16.0, Consul Template has the ability to maintain an arbitrary child process (similar to [envconsul](https://github.com/hashicorp/envconsul)). This mode is most beneficial when running Consul Template in a container or cgroup, particularly on a scheduler like [Nomad](https://www.nomadproject.io) or Kubernetes. When activated, Consul Template will spawn and manage the lifecycle of the child process.
```

Comment:
I would remove the bit on `or cgroup` and replace with `or on a cluster scheduler like...`